src/app/templates/ExpensesView.tsx

```diff
@@ -207,7 +206,7 @@ const ExpensesView: FC<ExpensesViewProps> = ({ expenses, estimates, mainnet, mod
         'relative rounded-md overflow-y-auto border',
         'flex flex-col text-gray-700 text-sm leading-tight'
       )}
-      style={{ height: '11rem' }}
+      style={{ height: modifyFeeAndLimit && modifyFeeAndLimit.totalFee <= MIN_GAS_FEE ? '10rem' : '11rem' }}
```

Comment:
move `modifyFeeAndLimit.totalFee <= MIN_GAS_FEE` into variable and reuse it

Reply:
you are using the same flag in the parent component (`InternalConfirmation ` `gasFeeError`).   
just pass it as a prop